webapp/src/store/index.js

```diff
@@ -263,6 +264,9 @@ export default createStore({
     setItemGraph(state, payload) {
       state.itemGraphData = payload;
     },
+    setBlocksInfos(state, blocksInfos) {
+      state.blocksInfos = blocksInfos;
```

Comment:
I think it would also be good if we can store this as a dict of block type -> block info, so that each block component can easily make use of its own info